_studio/mfx_lib/encode_hw/hevc/agnostic/base/hevcehw_base_legacy.cpp

```diff
@@ -4107,7 +4107,7 @@ bool CheckBufferSizeInKB(
     }
     else if (!bCqpOrIcq)
     {
-        mfxU32 frN, frD;
+        mfxU32 frN = 0, frD = 0;
```

Comment:
Since these are initialized by the std::tie function, I don't think you need to explicitly initialize them, but you can.  I would call this a KW false positive.